src/org/zaproxy/zap/utils/ZapSupportUtils.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2017 The ZAP Development Team
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.utils;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import org.parosproxy.paros.Constant;
+import org.zaproxy.zap.control.AddOn;
+import org.zaproxy.zap.control.AddOnLoader;
+import org.zaproxy.zap.control.ExtensionFactory;
+
+/**
+ * A class containing utility methods that can be used for support purposes.
+ * 
+ * @since TODO add version
+ */
+public class ZapSupportUtils {
+
+	public static String getProductName() {
+		return Constant.PROGRAM_NAME;
+	}
+
+	public static String getVersion() {
+		return Constant.messages.getString("support.version.label") + " " + Constant.PROGRAM_VERSION;

```

Comment:
The JavaDoc of the class could mention that the information returned is prefixed with labels.